spec/models/user_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+
```

Comment:
No user validations? 😞 

What about ensuring that emails are unique at least. That seems essential for the signup feature, else, if you deploy this, there could be multiple user records with the same email - not good.

Reply:
added user validation - not 100% on my spec for this though, if I use .build twice it doesn't work same goes for .create (or at least with .create my expectation must be wrong).